hack/e2e/run.sh

```diff
@@ -144,6 +144,15 @@ elif [[ "${CLUSTER_TYPE}" == "eksctl" ]]; then
   fi
 fi
 
+if [[ "${EBS_INSTALL_SNAPSHOT}" == true ]]; then
+  loudecho "Installing snapshot controller and CRDs"
+  kubectl apply --kubeconfig "${KUBECONFIG}" -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/v4.1.1/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml
```

Comment:
It's ok to be hardcoding the snapshot-controller version here?  Do we at least want to extract it into a variable to make it easier to change?

Reply:
fixed, made it variable